.goreleaser.yml

```diff
@@ -9,7 +9,8 @@ builds:
     binary: nibid
     hooks:
```

Comment:
Just a note, I believe setting the `$WASMVM_VERSION` in the `env` block won't be seen by the environment in the `pre` blocks. This should make `goreleaser` load whichever version is stated in our `go.mod`.